vscode/microsoft-kiota/src/kiotaInterop/types.ts

```diff
@@ -0,0 +1,257 @@
+export enum KiotaGenerationLanguage {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
shouldn't this file be in the lib section as well?

Reply:
it is